webapp/pages/notifications/index.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <ds-card space="small">
+    <ds-flex class="notifications-page-flex">
+      <ds-flex-item :width="{ lg: '85%' }">
+        <ds-heading tag="h3">{{ $t('notifications.title') }}</ds-heading>
+      </ds-flex-item>
+      <ds-flex-item width="110px">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-10-30T08:39:22Z" title="Wednesday, October 30th 2019, 9:39:22 am +01:00">Oct 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,106 @@
+<template>
+  <ds-card space="small">
+    <ds-flex>
+      <ds-flex-item :width="{ lg: '85%' }">
+        <ds-heading tag="h3">{{ $t('notifications.title') }}</ds-heading>
+      </ds-flex-item>
+      <ds-flex-item class="sorting-dropdown">
```


Here the styleguide messes things up again... If you don't give the `ds-flex-item` a `width` it will have a `flex-grow: 1` inline style so it will take up the whole remaining width of the flexbox, making it impossible to align it right.

As a workaround I would suggest:
1. give the `ds-flex-item` a `width="100px"`
2. remove the `sorting-dropdown` class
3. give the `ds-flex` a class with `justify-content: space-between;`

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-04T21:21:18Z" title="Monday, November 4th 2019, 10:21:18 pm +01:00">Nov 4, 2019</time>_

---


great trick!